modules/simple_forms_api/app/models/form_engine/employment_history.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module FormEngine
+  class EmploymentHistory
+    attr_reader :date_ended,
+                :date_started,
+                :highest_income,
+                :hours_per_week,
+                :lost_time,
+                :name_and_address,
+                :type_of_work
+
+    def initialize(data)
+      @data = data
+
+      set_ivars if data.present?
+    end
+
+    private
+
+    attr_reader :city, :country, :data, :name, :postal_code, :state, :street
+
+    def format_date(date)
+      date_arr = date.split('-')
+
+      "#{date_arr[1]}/#{date_arr[2]}/#{date_arr[0]}"
```

Comment:
What if any of the date array pieces are blank?

Reply:
The dates are a required field on the front-end and come to us in an XXXX-XX-XX format through the VADS pattern, so that should not be an issue. However, if for some reason they did, you were end up with a date string that looks like `XX//XXXX`, `//XXXX`, or `//`. While not aesthetically ideal, they would map fine to the PDF template and show that no data was entered for that field.